### PR TITLE
Added explanation for "Applied" option while creating new checkout term and condition

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added explanation for "Applied" option while creating new checkout term and condition

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

https://docs.magento.com/user-guide/sales/terms-and-conditions.html

### CODE REVIEW

File: src/sales/terms-and-conditions.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ _Terms and Conditions at checkout_
 
 1. Set **Status** to `Enabled`.
 
+1. Set **Applied** to `Automatically` or `Manually`.
+
+   |`Automatically`|Condition will be accepted automatically on checkout.|
+   |`Manually`|Customers are required manually accept created condition for placing an order on checkout page.|
```
Comment (dmytro-ch) [inline src/sales/terms-and-conditions.md:48]:
```suggestion
   |`Manually`|Customers are required to manually accept the created condition for placing an order on the checkout page.|
```

File: src/sales/terms-and-conditions.md [hunk 2]
```diff
@@ -42,6 +42,11 @@ _Terms and Conditions at checkout_
 
 1. Set **Status** to `Enabled`.
 
+1. Set **Applied** to `Automatically` or `Manually`.
+
+   |`Automatically`|Condition will be accepted automatically on checkout.|
```
Comment (dmytro-ch) [inline src/sales/terms-and-conditions.md:47]:
```suggestion
   |`Automatically`|Condition will be accepted automatically upon the checkout.|
```

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**